generators/base-application/index.ts

```diff
@@ -24,3 +24,4 @@ export { default } from './generator.js';
 export { default as comand } from './command.js';
 export type { Entity, Field, Relationship } from '../../lib/types/application/index.js';
 export type { Config, Options, Source } from './types.js';
+export type { AllConfig, AllOptions } from './types-all.js';
```

Comment:
In libs/*, shall disappear

Reply:
There is no downstream generator for this.